ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -39,6 +55,13 @@
                         (:client-id request-context)))
         save-subscription-result (ingest/save-subscription request-context concept-with-user-id)]
     ;; Log the successful ingest, with the metadata size in bytes.
+
+    ; there is a worry that subscriptions could be created neferiusly, log the
+    ; action. When no longer needed (log) also remove the variables from the let.
```

Comment:
This comment is worded strangely and is hard to make sense of

Reply:
how about:

;; In development developers pointed out that subscriptions could be created
;; neferiusly. To monitor this the action is logged so that we can decided
;; if a different approach is needed.  When the log is no longer needed,
;; also remove the variables from the let.